docs/deploying.md

```diff
@@ -0,0 +1,192 @@
+
+You will either need to configure the firewalls on each node in your cluster (recommended) to allow access to the cluster services or disable it completely.  Included in this section is configuration examples for `firewalld`.  If you use a different firewall, check your documentation for how to open the listed ports.
+
+Master node(s) require
+
+| Service Name | Port/protocol     |
+|--------------|-------------------|
+| zookeeper    | 2181/tcp          |
+| mesos-master | 5050/tcp          |
+| etcd         | 2379/tcp 4001/tcp |
+| marathon     | 8080/tcp          |
+
+Example `firewalld` config
+
+    $ sudo firewall-cmd --zone=public --add-port=2181/tcp --permanent
+    $ sudo firewall-cmd --zone=public --add-port=5050/tcp --permanent
+    $ sudo firewall-cmd --zone=public --add-port=2379/tcp --permanent
+    $ sudo firewall-cmd --zone=public --add-port=4001/tcp --permanent
+    $ sudo firewall-cmd --zone=public --add-port=8080/tcp --permanent
+    $ sudo systemctl restart firewalld
+
+Agent (compute) nodes require
+
+| Service Name | Port/protocol     |
+|--------------|-------------------|
+| BIRD (BGP)   | 179/tcp           |
+| mesos-agent  | 5051/tcp          |
+
+Example `firewalld` config
+
+    $ sudo firewall-cmd --zone=public --add-port=179/tcp --permanent
+    $ sudo firewall-cmd --zone=public --add-port=5051/tcp --permanent
+    $ sudo systemctl restart firewalld
+
+# Set up Master services.
+
+Pull the Docker images.  This will take a few minutes.
+
+    $ sudo docker pull jplock/zookeeper:3.4.5
+    $ sudo docker pull quay.io/coreos/etcd:v2.2.0
+    $ sudo docker pull spikecurtis/mesos-calico:0.25.0-rc2
+    $ sudo docker pull mesosphere/marathon:v0.9.1
+    $ sudo docker pull calico/node:v0.7.0
+
+
+    $ wget https://github.com/projectcalico/calico-mesos/releases/0.1/units.tgz
+
+## Zookeeper
+
+We'll use systemd to keep Zookeeper running.  Copy the [Zookeeper unit file](units/zookeeper.service) into `/etc/systemd/system/`
+
+
+    $ sudo cp zookeeper.service /usr/lib/systemd/system/
+    $ sudo systemctl enable zookeeper.service
+    $ sudo systemctl start zookeeper.service
+
+## Mesos Master
+
+    $ sudo cp mesos-master.service /usr/lib/systemd/system/
+    $ sudo systemctl enable mesos-master.service
+    $ sudo systemctl start mesos-master.service
+
+## Etcd
+
+`etcd` needs your fully qualified domain name to start correctly.  The included
+unit file looks for this value in `/etc/sysconfig/etcd`
+
+    $ sudo sh -c 'echo FQDN=`hostname -f` > /etc/sysconfig/etcd'
+    $ sudo cp etcd.service /usr/lib/systemd/system/
+    $ sudo systemctl enable etcd.service
+    $ sudo systemctl start etcd.service
+
+## Marathon
+
+    $ sudo cp marathon.service /usr/lib/systemd/system/
+    $ sudo systemctl enable marathon.service
+    $ sudo systemctl start marathon.service
+
+# Set up Agent Services
+
+## Calico
+
+    $ curl -L -O https://github.com/projectcalico/calico-docker/releases/download/v0.7.0/calicoctl
```

Comment:
calico-node-v0.7.0 uses libcalico-v0.3.0. The submodule should probably be updated in this PR to use that revision of libcalico as well